aws-android-sdk-s3/src/main/java/com/amazonaws/services/s3/internal/Constants.java

```diff
@@ -39,6 +39,11 @@
     /** Service hostname for accessing dualstack accelerated S3 buckets */
     public static final String S3_ACCELERATE_DUALSTACK_HOSTNAME = "s3-accelerate.dualstack.amazonaws.com";
 
+    /** Parameter name to indicate permission to dangerously connect to HTTP endpoint */
+    public static final String LOCAL_TESTING_FLAG_NAME = "DangerouslyConnectToHTTPEndpointForTesting";
+    /** Default endpoint for the local S3 storage testing */
+    public static final String LOCAL_TESTING_ENDPOINT = "http://10.0.2.2:20005";
```

Comment:
Add a comment here that this IP is used by Android emulators to connect to localhost.